btclightclient/partial_merkle_tree.go

```diff
@@ -0,0 +1,215 @@
+package btclightclient
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"errors"
+	"io"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+)
+
+// / We verify partial merkle tree proof. This logic are using in verifytxoutproof
+// / We are follow the logic in verifytxoutproof.
+
+type PartialMerkleTree struct {
+	numberTransactions uint
+	vBits              []bool
```

Comment:
please explain (in comment just above this line) what is the interpretation of `vBits` slice.

Reply:
Done!